Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.8
+
+LABEL maintainer "Bitnami <containers@bitnami.com>"
+
+ARG BKPR_VERSION
+
+RUN : "${BKPR_VERSION:?BKPR_VERSION build argument not specified}"
```

Comment:
dockerfile requires that the BKPR_VERSION is specified during the build, this line breaks the build when not specified